_config/mimetypes.yml

```diff
@@ -756,7 +756,6 @@ SilverStripe\Control\HTTP:
     stl: application/vnd.ms-pki.stl
     str: application/vnd.pg.format
     stw: application/vnd.sun.xml.writer.template
-    sub: image/vnd.dvb.subtitle
```

Comment:
I think we can as everything seems to have been working up to this point and there would have been an overwritten value anyway